virtManager/manager.py

```diff
@@ -171,6 +171,13 @@ def __init__(self):
         for conn in connmanager.conns.values():
             self._conn_added(connmanager, conn)
 
+        # Select the first item in the list if it exists
+        vmlist = self.widget("vm-list")
+        model = vmlist.get_model()
+        if len(model) > 0:
+            selection = vmlist.get_selection()
+            selection.select_path(Gtk.TreePath.new_first())
+
```

Comment:
This change is unrelated to the PR so it should not be part of it at all and I don't see any justification to make such change.